riscv/build.rs

```diff
@@ -29,6 +29,7 @@ fn build_instruction_tests() {
             .to_string()
             .strip_suffix(".S")
         {
+            println!("cargo:rerun-if-changed={file_name}.S");
```

Comment:
Did you mean to use `write!` instead of `println!`?

Reply:
No, this is a command to cargo rebuild if the assembly file changes.